mailpoet/tests/integration/Segments/DynamicSegments/Filters/WooCommerceNumberOfOrdersTest.php

```diff
@@ -2,15 +2,19 @@
 
 namespace MailPoet\Segments\DynamicSegments\Filters;
 
+use MailPoet\Segments\DynamicSegments\Filters\WooCommerceNumberOfOrders;
 use Helper\Database;
 use MailPoet\Entities\DynamicSegmentFilterData;
 use MailPoet\Entities\DynamicSegmentFilterEntity;
 use MailPoet\Entities\SegmentEntity;
 use MailPoet\Entities\SubscriberEntity;
+use MailPoet\Subscribers\Source;
+use MailPoet\Subscribers\SubscribersRepository;
 use MailPoetVendor\Carbon\Carbon;
 
 class WooCommerceNumberOfOrdersTest extends \MailPoetTest {
 
+  /* WooCommerceNumberOfOrders */
```

Comment:
Another small detail is that the annotation should be `/** @var WooCommerceNumberOfOrders */`.